src/styles/header.scss

```diff
@@ -0,0 +1,66 @@
+@use './variables' as *;
+
+.moyo-header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  box-sizing: border-box;
+  padding: 0 50px;
+  overflow: auto;
```

Comment:
Using `overflow: auto;` on the header might not be necessary unless you expect the header content to overflow. If the header is meant to be responsive, consider other responsive design techniques.